### PR TITLE
Fixing button centering and mor-info mobile layout.

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
@mmmavis Just fixing some button vertical centering and the more-info bit near the bottom had some not great mobile views.

File: source/sass/main.scss [hunk 1]
```diff
@@ -325,13 +327,17 @@ button {
...
+      min-height: 220px;
+
+      @media screen and (min-width: $bp-md) {
+        display: flex;
+        justify-content: space-around;
+        align-items: center;
+      }
     }
 
     .info-box {
```
Comment (mmmavis) [inline source/sass/main.scss:339]: I'd suggest to apply top and bottom padding to `.more-info .container` and `margin-bottom` to `.info-box` only if it's not the last child (e.g., `.info-box:not(:last-child)`) . If we just do `margin: 1.5rem 0;` on all `.info-box` the space in between `.info-box` is doubled than their outer margin.

**Review by mmmavis — CHANGES_REQUESTED** [review]

File: source/sass/main.scss (hunk 1 above)
Comment (ScottDowne) [reply source/sass/main.scss:339]: Actually, those margins should be overlapping. However, `info-box` is erroneously `display: inline-block;` which isn't actually being used for anything anymore (it was at one point, but I went with flexbox instead), and causing the margins to not overlap. If it's `display: block;` and properly cleared and centered, we can get the same result.

**Comment by mmmavis** [comment]
Since we are using `mofo-bootstrap`, a lot of the custom CSS here should be replaced by applying `mofo-bootstrap` classes to elements. 

Let's merge this PR first and work on cleaning up custom CSS on this project. Many of the `display` rules shouldn't be needed if we are using mofo-bootstrap effectively.

**Review by mmmavis — APPROVED** [review]

**Comment by ScottDowne** [comment]
@mmmavis whoops, I was fixing stuff around the margins, and didn't actually add the files to this commit, so they didn't get merged in.

I made another pull request here: https://github.com/mozilla/nsf-challenge/pull/6 which we can consider or not, based on what the display is like on mofo bootsrap.

I mostly just don't think of using mofo-bootstrap for things simply because the scope of what it can do is unfamiliar and abstract to me.

**Comment by mmmavis** [comment]
> I mostly just don't think of using mofo-bootstrap for things simply because the scope of what it can do is unfamiliar and abstract to me.

Heh yea that I understand but I would encourage you to start getting familiar with it as it would make our CSS code a lot easier to maintain. As now we have `mofo-bootstrap` on all (or most?) of our MoFo websites it's time to embrace it 😜

**Pull request merged**